substrate/scripts/run_all_benchmarks.sh

```diff
@@ -80,11 +81,7 @@ EXCLUDED_PALLETS=(
 
 # Load all pallet names in an array.
 ALL_PALLETS=($(
-  $SUBSTRATE benchmark pallet --list --chain=dev |\
-    tail -n+2 |\
-    cut -d',' -f1 |\
-    sort |\
-    uniq
+  $SUBSTRATE benchmark pallet --list=pallets --chain=dev
```

Comment:
Would still need a `--no-csv-header` option for this to work, since it otherwise prints `pallet` as first line.

Reply:
just out of curiosity, is there any reason to output it in a csv-friendly format?

Reply:
Dunno, but now its expected behaviour 🙃

Reply:
ok, I added `no-csv-header` option :)